docs/ingest-api-walkthrough.md

```diff
@@ -61,7 +61,15 @@ Evaluate the expression `_links.protocols.href` from the response to the previou
 >: curl -X PUT -H "Content-Type: text/uri-list" -d "http://api.ingest.integration.data.humancellatlas.org/protocols/5b769e991c022d00079f1d95" "http://api.ingest.integration.data.humancellatlas.org/processes/5b769e3d1c022d00079f1d92/protocols"
 ```
 
-## 7. Add a Bundle Reference to this Analysis
+## 7 Link input Files to the Analysis Process
+
+Any files in the DCP that were used as an analysis inputs should be linked to the analysis process. Given an input file UUID, this can be performed as follows:
+
+```bash
+>: curl -X POST -H "Content-Type: application/json" http://api.ingest.dev.data.humancellatlas.org/processes/{analysis_id}/add-input-file -d '{"inputFileReference": "<input file UUID>"}'
```

Comment:
Is it possible to use `inputFiles`, POST http://api.ingest.dev.data.humancellatlas.org/processes/{analysis_id}/inputFiles ?